CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ positions are near surface of whatever celestial body their positions are refere
 (either the Earth or Moon, currently).
 
 ### Changed
+- Improved performance for `antpair2ind` and `_key2inds` by using the
+`blts_are_rectangular` parameter, and also by caching the results. To improve performance of the cache, the resulting indices are returned as slices whenever possible.
```

Comment:
Minor quibble, but can you wrap this line to make it similar to the rest of the file?

Reply:
Ah, my editor wrapped it for me so I didn't notice. Thanks!